pkg/kritis/review/validating_transport_test.go

```diff
@@ -31,6 +31,10 @@ import (
 	"github.com/grafeas/kritis/pkg/kritis/util"
 )
 
+func enc(in string) string {
```

Comment:
Let's rename this to `to64b` or `encode64b` or something similar, so that it's more readable at the call sites without having to check the definition.

Reply:
Done.